install.sh

```diff
@@ -237,10 +246,27 @@ if [ $IONCUBE_INSTALLED = "no" ]; then
         service php5-fpm restart &> /dev/null
     fi
 
+    # If php-fpm php7 is used
+    if [ -d "/etc/php/$PHPVERSION/fpm/conf.d" ]; then
+        # debian/ubuntu
+        ln -s $PHPDIR/00-ioncube.ini /etc/php/$PHPVERSION/fpm/conf.d/00-ioncube.ini
+        # Restart php-fpm
+        # TODO: will eventually be php7.1-fpm
```

Comment:
I'm not yet sure on the bets way to handle this. Perhaps `php$PHPVERSION-fpm` might work for a command.